src/lib/Transport/index.ts

```diff
@@ -49,6 +48,17 @@ export default class Transport {
     );
   }
 
+  isMultiPartRequest(headers: IncomingHttpHeaders): boolean {
+    for (const header of Object.keys(headers)) {
```

Comment:
How about this?
```
   Object.entries(headers).some(([header, value]) => header.toLowerCase() === 'content-type' && value.toString().toLowerCase().includes('multipart'))
```

Reply:
Cool idea. I'll put it in!

Reply:
Haha maybe it's just harder to read now.

Oh well!